### ISSUE TITLE
 v1.55.1 tag not pushed to quay.io

### ISSUE THREAD

**Issue by mdbooth** [kubevirt/containerized-data-importer#2473]
**What happened**:
Following installation instructions from: https://kubevirt.io/user-guide/operations/containerized_data_importer/

Latest tag detected by scripts on that page is `v1.55.1`, which is the image referenced by the release assets.

On Installing CDI, the CDI fails to start with ErrImagePull:

```
  Warning  Failed     <invalid> (x4 over <invalid>)   kubelet            Failed to pull image "quay.io/kubevirt/cdi-operator:v1.55.1": rpc error: code = Unknown desc = reading manifest v1.55.1 in quay.io/kubevirt/cdi-operator: manifest unknown: manifest unknown
```

v1.55.0 works fine.

**Comment by akalenyu** [comment]
Seems the push job failed, https://storage.googleapis.com/kubevirt-prow/logs/push-release-containerized-data-importer-images/1593641423020756992/build-log.txt
@awels Any idea what went wrong here? should we rerun because we are definitely authorized

**Comment by awels** [comment]
I switched latest to 1.55.0 for now, once we get the images pushed, I will switch it back.

**Comment by awels** [comment]
Okay the job was fixed, and 1.55.1 now exists in quay. Put the latest back to 1.55.1

**Comment by awels** [comment]
Closing please reopen if not solved.
